src/main/java/net/mcreator/ui/modgui/LivingEntityGUI.java

```diff
@@ -825,6 +844,14 @@ private synchronized void regenerateAITasks() {
 			bossBarType.setEnabled(isBoss.isSelected());
 		});
 
+		mobBehaviourType.addActionListener(e -> {
+			spawnInRaids.setEnabled(mobBehaviourType.getSelectedItem() == "Raider");

```

Comment:
Merge with master and handle those enabled/disabled states in enableOrDisableFields so other actions don't override enabled/disabled states already defined

Reply:
Ok

Reply:
Done